src/SumoCoders/FrameworkCoreBundle/Mail/MessageFactory.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Mail;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use TijsVerkoyen\CssToInlineStyles\CssToInlineStyles;
+
+class MessageFactory
+{
+    /**
+     * @var array
+     */
+    protected $sender = array();
```

Comment:
Why should it? The open closed principle states that "software entities (classes, modules, functions, etc.) should be open for extension, but closed for modification".

You now open your class for modification, so everyone can extend it and start to do stupid things with it.

I think it's a better default to make classes not extendable. If you want to extend them, it's still possible by changing the properties you want to extend. IMO, making properties private and classes final is a sane default in most cases.